src/routes/docs/contributing/testing/+page.md

```diff
@@ -12,8 +14,170 @@ We test components with Storybook's "Interaction" tests for our integration of S
 
 We test the fully running artifact with Playwright. Testing the widget is done against a running Svelte app that imports the built package of the Widget as if installed via NPM. Tests are in the root `tests` directory.
 
+### Manual tests
+
+These are end-to-end tests that cannot be automated and must be manually tested. This usually involves randomly generated codes that reside outside of the system, and are hard or impossible to mock.
+
+#### Please manually test the following
+
+1. Journey continuation feature (e.g. Email Suspend Node) requires manual testing due to the randomly generated code sent to email which cannot be automated at this time.
```

Comment:
Noting that this will be a manual test, at least until we develop a mock server that could help automate this.